pkg/logs/scheduler/scheduler.go

```diff
@@ -116,41 +182,53 @@ func (s *Scheduler) Unschedule(configs []integration.Config) {
 					s.sources.RemoveSource(source)
 				}
 			}
-		case s.newService(config):
-			// new service to remove
-			entityType, _, err := s.parseEntity(config.TaggerEntity)
-			if err != nil {
-				log.Warnf("Invalid service: %v", err)
-				continue
-			}
-			// logs only consider container services
-			if entityType != containers.ContainerEntityName {
-				continue
-			}
-			log.Infof("New service to remove: entity: %v", config.Entity)
-			service, err := s.toService(config)
-			if err != nil {
-				log.Warnf("Invalid service: %v", err)
-				continue
-			}
-			s.services.RemoveService(service)
```

Comment:
same thing for [this](https://github.com/DataDog/datadog-agent/blob/7.32.x/pkg/autodiscovery/autoconfig.go#L660)